src/actions/blogActions.test.js

```diff
@@ -0,0 +1,69 @@
+import { ADD_POST, addPost, deletePost, UPDATE_POST, updatePost, DELETE_POST } from './blogActions';
+const Chance = require('chance');
+
+describe('blog actions', () => {
+  it('creats an ADD_POST action', () => {
+    const action = addPost({
+      title: chance.name(),
```

Comment:
In order to use the chance library like that you also have to include a new instance of chance. It should look something like this
```js
  const chance = new Chance();
  it('creats an ADD_POST action', () => {
    const action = addPost({
      title: chance.name(),
      body: chance.sentence({ words: 10 })
    });
```